src/pages/Reader.tsx

```diff
@@ -113,9 +111,11 @@ class Reader extends React.Component<any, State> {
       return;
     }
     const pages = await getPages(this.state.mangafile, chapters[this.state.chapter].name);
+    const metadata = await getMetadata(this.state.mangafile);
```

Comment:
what happens if this fails (ie `.metadata` is missing for some reason) - should prob display a popup that says manga failed to load or sm, same thing with `getPages`

Reply:
I feel like that's unnecessary, too much work

Reply:
but i dont think the app should die if something goes wrong - u can do sm small like a popup banner, or just log it in console and stay on the main page

Reply:
You want a user to get a popup if a description is missing? I will definitely have checks so I don't display undefined but that's about it.

Reply:
no not missing, if the metadata or manga pages fail to load for whatever reason

Reply:
ie, it's missing, the file server is down or whatever

Reply:
the point is that the carrot app shouldn't completely die if the backend (file server) does something wrong

Reply:
Oh ok